### PR TITLE
enh(log_v2): remove old logs REFS: MON-7362

### PR DESCRIPTION
**Pull request by rem31**
## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: core/src/misc/diagnostic.cc [hunk 1]
```diff
@@ -260,13 +233,13 @@ void diagnostic::generate(std::vector<std::string> const& cfg_files,
...
-                                    << "' parsing failed: " << e.what();
+      log_v2::core()->error(
+          "diagnostic: configuration file '{}' parsing failed: {}", *it,
+          e.what());
     }
 
     // ls.
-    logging::info(logging::high)
-        << "diagnostic:     getting modules information";
+    log_v2::core()->info("diagnostic:     getting modules information");
```
Comment (centreonpm) [inline core/src/misc/diagnostic.cc:242]: strange spaces

File: lua/src/macro_cache.cc [hunk 2]
```diff
@@ -547,11 +542,10 @@ void macro_cache::_process_service_group(
...
   auto const& sgm = std::static_pointer_cast<neb::service_group_member>(data);
-  logging::debug(logging::medium)
-      << "lua: processing service group member "
-      << " (group_name: '" << sgm->group_name
-      << "', group_id: " << sgm->group_id << ", host_id: " << sgm->host_id
-      << ", service_id: " << sgm->service_id << ")";
+  log_v2::lua()->debug(
+      "lua: processing service group member (group_name: {}, group_id: {}, "
+      "service_id: {}",
+      sgm->group_name, sgm->group_id, sgm->host_id, sgm->service_id);
```
Comment (centreonpm) [inline lua/src/macro_cache.cc:548]: you forgot host_id in the text

File: neb/src/initial.cc [hunk 3]
```diff
@@ -191,15 +191,15 @@ static void send_host_dependencies_list() {
       neb::callback_dependency(NEBCALLBACK_ADAPTIVE_DEPENDENCY_DATA, &nsadd);
     }
   } catch (std::exception const& e) {
-    logging::error(logging::high)
-        << "init: error occurred while dumping host dependencies: " << e.what();
+    log_v2::neb()->info(
+        "init: error occurred while dumping host dependencies: ", e.what());
```
Comment (centreonpm) [inline neb/src/initial.cc:195]: {} is missing

File: neb/src/neb.cc [hunk 4]
```diff
@@ -167,47 +138,34 @@ int nebmodule_init(int flags, char const* args, void* handle) {
...
     neb::gl_registered_callbacks.push_back(std::shared_ptr<neb::callback>(
         new neb::callback(NEBCALLBACK_PROCESS_DATA, neb::gl_mod_handle,
                           &neb::callback_process)));
     neb::gl_registered_callbacks.push_back(
         std::shared_ptr<neb::callback>(new neb::callback(
             NEBCALLBACK_LOG_DATA, neb::gl_mod_handle, &neb::callback_log)));
-
   } catch (std::exception const& e) {
-    logging::error(logging::high) << "main: cbmod loading failed: " << e.what();
+    log_v2::core()->error("main: cbmod loading failed: ", e.what());
```
Comment (centreonpm) [inline neb/src/neb.cc:163]: {} is missing

**Review by bouda1 — CHANGES_REQUESTED** [review]

**Review by bouda1 — APPROVED** [review]

**Pull request merged**